spec/features/profile_photos_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "profile photo" do
+  scenario "user uploads a profile photo" do
+    visit root_path
+    click_link "Sign Up"
+
+    fill_in "Email", with: "ash@s-mart.com"
+    fill_in "Password", with: "boomstick!3vilisd3ad"
+    fill_in "Password confirmation", with: "boomstick!3vilisd3ad"
+    attach_file :user_profile_photo, "#{Rails.root}/spec/support/images/puppies.jpg"
+    click_button "Sign up"
+
```

Comment:
You'll need to test that the image has appeared on the page. Both the carrierwave docs and the reading should show you how to do this